tests/tpm-vm

```diff
@@ -45,7 +45,7 @@ lxc stop "${vmName}" --force
 
 # TPM names are included on the swtpm socket path and long socket paths can cause problems if not handled correctly.
 echo "==> Test handling TPMs with long names"
-longName="tpm-device-with-long-name-for-testing"
+longName="device-with-very-long-name-and-/-4-qemu-property-handling-test_"
 # XXX: LXD releases 5.21 and earlier don't support long names (yet)
```

Comment:
Is the comment about LXD 5.21 and below still applicable? I didn't quite follow the stream of backports.

Reply:
I think so, although it should be possible to support TPM long names without breaking anything, unlike disk passthrough devices.